src/General/Typography/Paragraph.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import Paragraph from './Paragraph';
+import { PrimaryColor } from '../../Style/Colors';
+
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+const paragraphText =
+  'As the first sign in the zodiac, the presence of Aries always marks the beginning of something energetic and turbulent. They are continuously looking for dynamic, speed and competition, always being the first in everything - from work to social gatherings.';
+
+describe('displays the correct text for each variant:', () => {
+  const variants: any[] = ['subtitle', 'regular', 'caption', 'smallest'];
+  variants.forEach(variant => {
+    it(`${variant}`, () => {
+      const { getByText } = render(
+        <Paragraph variant={variant}>{paragraphText}</Paragraph>
+      );
+      const paragraphTag = getByText(paragraphText);
+      expect(paragraphTag).toHaveTextContent(paragraphText);
```

Comment:
Because `getByText` matches full strings and case-sensitive by default,  `toHaveTextContent` seems to be redundant.
Can use `expect(paragraphTag).toBeTruthy()` or `expect(paragraphTag).toContainHTML('p')` to instead.

Reply:
I think we should add testing for `font-size` here

```
const fontSize: number[] = [18, 16, 14, 12];
...
const size = fontSize[index];
expect(paragraphTag).toHaveStyle(`font-size: ${size}px`);
```